### ISSUE TITLE
Bug filtering input files

### ISSUE THREAD

**Issue by jvarada** [poldracklab/mriqc#643]
Cmd:
```
mriqc --verbose-reports --write-graph --no-sub --nprocs 32 
--mem_gb 121 --participant_label 0089 --session-id 02 
--run-id 04 --modalities bold --ica
/gpfs/gsfs7/users/varadajc/mriqc_summaries/3Tc/2017_05/bids_2017_05_3Tc
/gpfs/gsfs7/users/varadajc/mriqc_summaries/3Tc/2017_05/mriqc_results 
participant
```

Got:
```
RuntimeError: The combination of supplied labels 
(--participant_label null 0089 null) (--session-id 02) (--run-id 04) 
did not result in matches within the BIDS directory
(/gpfs/gsfs7/users/varadajc/mriqc_summaries/3Tc/2017_05/bids_2017_05_3Tc).
```

Confirmed the specified file exists in:

bids_dir/
----sub-0089/
--------ses-02/
------------func/
----------------sub-0089_ses-02_task-NoTaskSpecified_run-04_bold.json
----------------sub-0089_ses-02_task-NoTaskSpecified_run-04_bold.nii.gz

**Comment by jvarada** [comment]
I'll look into this. Not sure if it has to do with the addition of 'null' before/after single item spec in the BIDS file getter or if it's trying to filter subjects before filtering modalities (the 'anat' folder did not have a session 4).

**Comment by lnnrtwttkhn** [comment]
Hello, has this issue been resolved yet? I'm currently running into the very same problems when running mriqc on multiple functional runs: I confirmed that the specified files exist but still get the same error message as @jvarada described above. Thanks in advance for your help!

**Comment by oesteban** [comment]
Fixed in #735
